src/main/java/com/google/devtools/build/lib/remote/disk/DiskCacheClient.java

```diff
@@ -102,13 +111,57 @@ public ListenableFuture<Void> downloadBlob(
         MoreExecutors.directExecutor());
   }
 
+  private void checkDigestExists(Digest digest) throws CacheNotFoundException {
+    if (digest.getSizeBytes() == 0) {
+      return;
+    }
+
+    if (!toPath(digest.getHash(), /* isActionCache= */ false).exists()) {
```

Comment:
The name of the boolean parameter is `actionResult`. (I agree that `isActionCache` is better, but in that case we should also rename other instances.)

Reply:
Done.